ui/tests/playwright/BrokerPropertyPage.test.tsx

```diff
@@ -6,7 +6,7 @@ describe("Brokers property page", () => {
     await page.goto(
       `${URL}/kafka/j7W3TRG7SsWCBXHjz2hfrg/nodes/0/configuration`
     );
-    await page.waitForLoadState("networkidle");
+    await page.waitForLoadState("networkidle",{timeout: 60000});
```

Comment:
Why not setting it as the default timeout instead of overriding each wait? https://playwright.dev/docs/test-timeouts

Also, why is this needed? The default of 30s should be plenty.